template/apps/reactive-resume.json

```diff
@@ -4,22 +4,60 @@
 		"Tools"
 	],
 	"description": "A one-of-a-kind resume builder that's not out to get your data. Completely secure, customizable, portable, open-source and free forever.",
-	"image64": "martadinata666/reactive-resume:latest",
 	"logo": "https://raw.githubusercontent.com/pi-hosted/pi-hosted/master/images/reactiveresume.png",
 	"name": "reactive-resume",
-	"officialDoc": "https://hub.docker.com/r/martadinata666/reactive-resume",
+	"officialDoc": "https://hub.docker.com/r/martadinata666/reactive-resume, or read docs/reactive-resume.md",
```

Comment:
This line has to be only the address, otherwise the link will not work and will generate errors. You're already adding this link through piHostedDoc